Cargo.toml

```diff
@@ -18,6 +18,6 @@ once_cell = "1"
 
 [dev-dependencies]
 assert_cmd = "2"
-reqwest = { version = "0.11.10", features = ["blocking"] }
+reqwest = { version = "0.11.11", features = ["blocking"] }
```

Comment:
Could we lock it to only the minor version and not the patch? We should probably do this for all crates so that we can pull the patches automatically with `cargo update`.

`reqwest = { version = "0.11", features = ["blocking"] }`

Reply:
It doesn't make a lot of difference tbh.  Specifying 0.11.11 will still cause `cargo update` to update the patch level.  All it does is make 11 the minimum patch version (e.g. 0.11.10 is no longer acceptable, but it would be if we specified `0.11` in the manifest).